sycl/test/regression/pi_opencl_symbol_check.txt

```diff
@@ -0,0 +1,42 @@
+piContextCreate
```

Comment:
What's that for?

Reply:
Are you asking why I am checking this symbol from the export symbol list?
This is one of the symbols defined in pi_opencl.cpp.
Any particular reason why we should not export?

Reply:
@alexbatashev , I am sorry, could you be more specific about your change request?
The main purpose of this file is to check the exported symbols in libpi_opencl.so library.
The test code itself is test/regression/pi_opencl_symbol_check.cpp that I implemented as a part of this PR.
This test was requested by @smaslov-intel to avoid regression in the future.

Reply:
@bso-intel sorry, let me clarify. This file is not a test at all. It’s missing run line, and it has the wrong postfix. Only .cpp and .dump files are processed by lit. Have a look at how these tests implemented in abi directory. Also, this test would feel more comfortable in the abi directory.

Reply:
@alexbatashev ,
Exactly. This .txt file is just used as a snapshot record of currently exported pi* symbols.
As I mentioned, the LIT regression test code is pi_opencl_check.cpp, and this .txt file is used for comparison purposes.
I know that libsycl.so uses test/abi/sycl_symbols.linux.dump file which mixed data and code into a single file.
I just split them into two files, one for the code and the other for the database.
Do you want me to follow the single .dump file and move it under test/abi/?

Reply:
@bso-intel yes, let’s follow a common pattern.

Reply:
@alexbatashev , Sure, I just pushed the change you requested.